app/common/util/validationUtil.js

```diff
@@ -3,7 +3,7 @@
 Bahmni.Common.Util.ValidationUtil = (function () {
```

Comment:
@larslemos is this util being used in the app? It came with bahmni `common` but I can't find anywhere in use, I'm not even sure we'll need it in the future. I think we should give priority to test the code that we created or changed and is being used.